### DISCUSSION TITLE
Are tests failing?

### DISCUSSION THREAD

**Discussion by tuturto** [hylang/hy#2439] [Q&A] [answered]
I followed instruction at [hacking on Hy](https://docs.hylang.org/en/master/hacking.html). After the test run, there were 39 failed, 508 passed and 1 xfailed tests. I'm running Python 3.10.7 on Lubuntu. Is this a common problem or do I just have something wrong with my environment?

**Comment by Kodiologist** [comment]
Hey buddy, it's been a few.

Yeah, something is probably wrong on your end. You can always see the latest run on GitHub Actions here: https://github.com/hylang/hy/actions

That said, `hacking.rst` hasn't been updated in a while and has some obsolete stuff on it. I removed support for `tox` years ago, for example.

**Comment by tuturto** [comment]
Indeed, too many years even.

Ok, good to know it's just me. Maybe I can see what the problem is and update `hacking.rst` accordingly.

**Reply by Kodiologist** [reply, marked as answer]
I think I'll submit a PR later today (edit: #2440) to remove a lot of obsolete or unnecessary stuff from `hacking.rst`. In the meantime, `pip install -e . && pip install pytest && pytest` should suffice, modulo some deactivation and reactivation of your virtualenv to make sure you're running the new `hy` and `pytest`.

**Reply by tuturto** [reply]
This time it worked, although I was pretty confident I had done that already. Thank you for your help!
